test/e2e/javascript.spec.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  'Basic JavaScript test' : function (client) {
+    client
+      .url(client.launch_url)
+      .waitForElementVisible('#panels a:nth-child(3)', 1000)
+      .click('#panels a:nth-child(3)')
+      .waitForElementVisible('.javascript .CodeMirror', 1000)
+      .execute(function (text){
+        return $('.javascript .CodeMirror')[0].CodeMirror.setValue(text);
+      }, ["document.body.innerHTML = 'Hello'"])
+      .frame(2)
```

Comment:
Selecting a frame by name seems a bit scary here, e.g. if the ad is not shown, the frame number would change. 

We need a better way to identify frames.